src/main/java/org/opentripplanner/netex/mapping/StopAndStationMapper.java

```diff
@@ -92,7 +91,8 @@ void mapParentAndChildStops(final Collection<StopPlace> stopPlaces) {
         // were deleted in never versions of the StopPlace
         for (StopPlace stopPlace : stopPlaceAllVersions) {
             for (Quay quay : listOfQuays(stopPlace)) {
-                addNewStopToParentIfNotPresent(quay, station, fareZones, transitMode);
+                addNewStopToParentIfNotPresent(
+                        quay, station, fareZones, transitMode, selectedStopPlace);
```

Comment:
The question is should we move the mapping of `fareZones` and `transitMode` from above to `StopMapper`, or should we calculate the default `WheelChairBoarding` here and pass it down. I'd think it would be nicer if all of the three would be computed in the same place, as they are used in all of the child stops in a similar manner.

Reply:
@hannesj Now I understand exactly what you  mean, moved the business logic as requested in second commit.

Reply:
Thanks! Sorry for my poor initial explanation.